ogc/wms-layer-url/README.md

```diff
@@ -1,18 +1,33 @@
-# WMS Layer URL
+# WMS layer URL
 
-Display a WMS layer.
+Display a WMS layer using a WMS service URL.
 
-![](WmsLayerUrl.png)
+![Image of WMS layer URL](WmsLayerUrl.png)
+
+## Use case
+
+WMS is an OGC standard for displaying maps from images that are dynamically-generated on a web server. WMS is particularly useful for data that changes frequently, contains cartographically complex detail, or requires an open source data standard.
+
+## How to use the sample
+
+The map will load automatically when the sample starts.
 
 ## How it works
 
-To create a `WmsLayer` from a URL and display it:
+1. Create a `WmsLayer` specifying the URL of the service and the names of layers you want to display.
+    * **Note**: The name comes from the `Name` property, not the `Title` property. On many services, the title is human-readable while the name is a numeric identifier.
```

Comment:
we should avoid using backticks on non API methods/classes, suggest the following
```suggestion
    * **Note**: The name comes from the 'Name' property, not the 'Title' property. On many services, the title is human-readable while the name is a numeric identifier.
```